documentation.md

```diff
@@ -0,0 +1,14 @@
+# Documentation
+This guide aims to make it clear what information can and cannot be included in documents and designs that are made public.
+
+## Restricted Items
+1. Network ranges and IP addresses <br>
+The organisation currently have these classed as _sensitive_
+1. S3 Bucket names <br>
+Bucket names for a public facing endpoint. Bucket names are typically a hash to keep them obscured, putting them in a document would give context and remove the ambiguity.
+2. Individuals names, contact details or any other personally identifiable information.
+
+## Standards
+1. Documentation should be published in Markdown language
+1. Diagrams should be embedded as images, with a link to the image for easy of viewing
```

Comment:
We should be clear about our policy on whether diagrams are suitable to live in github.com. For architectural patterns it's straightforward - these should be available on github.com. For actual designs, I would argue that all of our designs are sufficiently vanilla that they should also be on github.com either as-is, or with a minimal effort to turn these into architectural patterns that others can use verbatim. But we'll need to distill a set of formal principles around this.

Reply:
if the don't contain any of the restricted items then don't they become sufficiently _vanilla_ ?